### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Croydon**
This PR restructures the readme.

* small fix: AppVeyor badget now shows status of the master branch instead of the latest push across all branches
* removed Sauerbraten features section: we are not Sauerbraten and have refered to our wiki for our own features all the way along, we also certainly didn't convice anyone to try out Inexor with that
* renamed "First steps" (What first steps for what purpose?? Not clear at all) to "Join us" and moved a few other information into this section
* fixing URLs

I will merge this Sunday without vetos.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -45,41 +44,41 @@ giving a general but no inevitably defined idea of where we're going next
...
+You have already accomplished the first step by reading this readme. Congratz!
 
-Mainly it is:
+The second one is only slightly harder by joining us in IRC (again see the wiki site for that: [Contact](https://github.com/inexor-game/code/wiki/Contact)) and idling/participating in our chat.
 
-1. fetching our needed build dependencies (cmake, git)
-2. downloading the code and data repository using git
-3. creating your specific build-files (makefile/Visual Studio-solution/XCode project/..) using CMake
-4. and lastly, building Inexor
+To contribute to the project and merge your changes into the master branch you need to convince enough members that the changes are an improvement.
```
Comment (a-teammate) [inline readme.md]: Maybe reformulate this to be less negative? "Convince enough members" sounds like a big job.
"it wont be an issue" sounds like there have been issues before.

We preferably should create a wiki page ("How do I dive into Inexor?") which could give general strategies for diving into our development process with different entry points.
(Best would be in a flow graph like manner: [Are you already familiar with the Cube Codebase?] -> [Yes/No] -> [Do you already have programming skills]..
People love graphs.

We furthermore need to convince people to not being afraid to "sound stupid" and "not being good enough".

File: readme.md [hunk 2]
```diff
@@ -88,30 +87,9 @@ Good tutorials on the whole process needed to build Inexor can be found in its [
...
- * Rendering engine optimized for high geometry throughput, supporting hardware occlusion culling and software precomputed conservative PVS with occluder fusion.
- * Lightmap based lighting with accurate shadows from everything including mapmodels, smooth lighting for faceted geometry, and fast compiles. Soft shadowmap based shadows for dynamic entities.
- * Pixel and vertex shader support, each model and world texture can have its own shader assigned. Supports normal and parallax mapping, specular and dynamic lighting with bloom and glow, environment-mapped and planar reflections/refractions, and post-process effects.
- * Loading of md2/md3/md5/obj/smd/iqm models for skeletal and vertex animated characters, weapons, items, and world objects. Supports animation blending, procedural pitch animation, and ragdoll physics for skeletally-animated characters.
- * Simple stereo positional sound system.
- * Particle engine, supporting text particles, volumetric explosions, soft particles, and decals.
- * 3d menu/gui system, for in-world representation of choices.
```
Comment (a-teammate) [inline readme.md]: +1 we really dont need to talk about all that.

File: readme.md (hunk 1 above)
Comment (Croydon) [inline readme.md]: I was thinking about something like that before, but imagined an interactive version on our website. Anyway sounds like work for another time. We also have How To Contribute Code/Content pages and a page which contains what to contribute.

**Comment by Croydon** [comment]
It has the same base and I cherry-picked but the hash changed and now it is not counted as merged?
Okay..

**Comment by Croydon** [comment]
Merged with 1d69190905b3e0099da6533a708dae99be0ff56a

**Pull request closed without merging**